addons/sale_mrp/models/sale_order_line.py

```diff
@@ -130,7 +130,7 @@ def _get_qty_procurement(self, previous_product_uom_qty=False):
         # Specific case when we change the qty on a SO for a kit product.
         # We don't try to be too smart and keep a simple approach: we use the quantity of entire
         # kits that are currently in delivery
-        bom = self.env['mrp.bom']._bom_find(self.product_id, bom_type='phantom')[self.product_id]
+        bom = self.env['mrp.bom'].sudo()._bom_find(self.product_id, bom_type='phantom')[self.product_id]
```

Comment:
I think you'll probably have to add in some extra parameters to make sure you don't get incorrect BoMs, i.e. make sure it's the correct company

Reply:
I changed it now. Thank you for the feedback!